pypeit/find_objects.py

```diff
@@ -1244,6 +1244,10 @@ def global_skysub(self, skymask=None, update_crmask=True, trim_edg=(0,0),
         correction using the sky spectrum, if requested. See Reduce.global_skysub()
         for parameter definitions.
         """
+        if self.par['reduce']['findobj']['skip_skysub']:
```

Comment:
This should be upstream in the control flow where it is called. It is odd to call a skysub routine and then return zero behind the scenes. Better would be to just put this skip upstream so the user can see that sky subtraction is being skipped in the main control flow.